schemas/README.md

```diff
@@ -0,0 +1,2 @@
+# schemas
+Schemas for various PrairieLearn JSON files. They are designed to be published independently of PrairieLearn for consumption by other tools or applications. They are published on npm in the `@prairielearn/schemas` package.
```

Comment:
@nwalters512 could you extend the README to note the procedure to update the schemas on `npm`?